parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -284,7 +284,11 @@ write_files:
             "mtu": 1500,
             "addIf": "eth0",
             "isGateway": true,
+            {{- if IsIPMasqAgentEnabled}}
+            "ipMasq": false,
+            {{- else}}
             "ipMasq": true,
+            {{- end}}
```

Comment:
forgive me if this is a too stupid question - is there no similar configuration for AzureCNI case? is it expected by containerd that AzureCNI (and any other network plugins) should take care of the ipmasq , or is it because it expect some dedicated agent like ipmasq-agent to do so?